<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>1435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>731</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dados</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2221</width>
      <height>1361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Dados</string>
     </attribute>
     <widget class="QTableWidget" name="tabela_dados">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>901</width>
        <height>321</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>12</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>112</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>29</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>43</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Continente</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>País</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IDH</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PIB</string>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Anos Medios</string>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Anos Esperados</string>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Expectativa de Vida</string>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="comboContinente">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>651</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>&lt;Selecionar Continente&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Africa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>América Central</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>América do Norte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>América do Sul</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Asia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Europa</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Oceania</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboPaises">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>651</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>&lt;Selecionar País&gt;</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_vida">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_medio">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_esperado">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_pib">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>651</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Anos Esperados de Escolaridades</string>
         </property>
         <property name="indent">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="anos_esperados">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>651</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>PIB</string>
         </property>
         <property name="indent">
          <number>91</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pib">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>651</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Anos Médios de Escolaridades</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="anos_medios">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>651</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Expectativa de Vida</string>
         </property>
         <property name="indent">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="expectativa">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>691</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="OrdenarPib">
         <property name="text">
          <string>Ordenar por Expectativa de vida</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OrdenanrIDH">
         <property name="text">
          <string>Ordenar por IDH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OrdenarExpectativa">
         <property name="text">
          <string>Ordenar por PIB</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OrdenarPais">
         <property name="text">
          <string>Ordenar por País</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OrdenarContinente">
         <property name="text">
          <string>Ordenar por Continente</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>510</y>
        <width>130</width>
        <height>54</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="OrdenarMed">
         <property name="text">
          <string>Ordenar Anos Médios</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Ordenar Anos Esperados</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QTableWidget" name="tabela_africa_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>País</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IDH</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Rounded MT Bold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lista de Países por continente</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>África</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3 melhores IDH</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela_africa_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>País</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IDH</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>3 piores IDH</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIDH">
    <property name="title">
     <string>IDH</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuIDH"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
